appcues/src/main/java/com/appcues/ui/ExperienceRenderer.kt

```diff
@@ -131,7 +133,7 @@ internal class ExperienceRenderer(override val scope: AppcuesScope) : AppcuesCom
         return newExperience.priority == NORMAL && state !is IdlingState
     }
 
-    suspend fun show(qualificationResult: QualificationResult) {
+    suspend fun show(qualificationResult: QualificationResult) = withContext(Dispatchers.Main) {
```

Comment:
an important spot here where `show(..)` can be called during network response handling, and if we are using a test CoroutineScope, we still have to ensure that it gets back to the Main thread here for the UI work. This is not an issue in normal usage, as the AppcuesCoroutineScope is already using Dispatchers.Main